src/data/director/navigation.tsx

```diff
@@ -1,6 +1,30 @@
 export const links = [
   {
-    name: "Interviews",
-    link: "/admin/interviews",
+    name: "Director",
+    link: "",
+  },
+  {
+    name: "Applications",
+    link: "/director/:program/application",
```

Comment:
if the link is dynamic, we should instead make each link a function which returns a string for the dynamically generated url